src/configValidator.js

```diff
@@ -0,0 +1,64 @@
+import logger from './logger';
+
+function isValid(missingConfigFields) {
+  if (missingConfigFields.length > 0) {
+    logger.info(
+      'configValidator',
+      `❗️Please add missing mandatory fields to your config: \n${missingConfigFields.toString()}`
```

Comment:
Add space between emoji and the word please. It looks slightly off on the reporting